longevity_test.py

```diff
@@ -318,7 +318,7 @@ def test_user_batch_custom_time(self):
         templated_table_counter = itertools.count()
 
         if customer_profiles:
-            cs_duration = self.params.get('cs_duration', default='50m')
+            cs_duration = self.params.get('cs_duration', default='1440m')
```

Comment:
why doing the change here ? in not in the scenario yaml.
if you are at it, please remove the default from here.